### ISSUE TITLE
Huawei P20 lite[ANE] overlay doesnt work

### ISSUE THREAD

**Issue by washinson** [phhusson/vendor_hardware_overlay#108]
No automatic brightness option and no cutout.
Tested all pie GSIs. I dont know how to fix it, but I can test overlay on my device.

PS. ANE hasnt ro.hw.oemName prop. Adding it in vendor doesnt fix problem.
Adb logs if need
[error.txt](https://github.com/phhusson/vendor_hardware_overlay/files/3194952/error.txt)
[all.txt](https://github.com/phhusson/vendor_hardware_overlay/files/3194953/all.txt)

**Comment by washinson** [comment]
With EMUI9 overlay doesnt work
With EMUI8 overlay works

**Comment by phhusson** [comment]
Probably fixed by https://github.com/phhusson/device_phh_treble/commit/9b40bccd829887d18d7a178953ac9dcc8ec97e45

**Comment by GodcraftPlata** [comment]
Extract the overlay files from lastest stock firmware

**Comment by washinson** [comment]
> 
> 
> Probably fixed by [phhusson/device_phh_treble@9b40bcc](https://github.com/phhusson/device_phh_treble/commit/9b40bccd829887d18d7a178953ac9dcc8ec97e45)

rw-system.sh doesnt start on startup

```
phhgsi_arm64_ab:/ $ getprop persist.sys.vndk

phhgsi_arm64_ab:/ $ getprop sys.usb.ffs.aio_compat

phhgsi_arm64_ab:/ $ 
```

> 
> 
> Extract the overlay files from lastest stock firmware

How can i do this?

**Comment by GodcraftPlata** [comment]
Extract lastest framework-res from stock and make the overlay.

**Comment by washinson** [comment]
I followed phhusson's idea and added ro.hw.oemName=ANE-L21 into build.prop.
(getprop ro.boot.product.hardware.sku = ANE-L21)
Overlay started up.

**Comment by washinson** [comment]
> Probably fixed by [phhusson/device_phh_treble@9b40bcc](https://github.com/phhusson/device_phh_treble/commit/9b40bccd829887d18d7a178953ac9dcc8ec97e45)

Sorry for incorrect information. Adb cleared the execute permission. Fix works. Thank you.
